### PR TITLE
Extend several RPC parameters needed in container scenarios

### PR DESCRIPTION
**Pull request by ls-ggg**
Extend several RPC parameters needed in container scenarios, including: freeze_cgroup, cgroups_mode, and cgroup_props.

### CODE REVIEW

**Comment by ls-ggg** [comment]
@adrianreber Can you merge my pull request?

**Comment by adrianreber** [comment]
Can you open a PR to fix the CI failure?

**Comment by adrianreber** [comment]
See #18 for the clippy fixes.

**Comment by adrianreber** [comment]
Please rebase.

File: src/lib.rs [hunk 1]
```diff
@@ -27,6 +54,10 @@ pub struct Criu {
     file_locks: Option<bool>,
     manage_cgroups: Option<bool>,
     work_dir_fd: i32,
+    freeze_cgroup: Option<String>,
+    cgroups_mode: Option<CgMode>,
+    cgroup_props: Option<String>,
+
```
Comment (adrianreber) [inline src/lib.rs:60]: Please remove this empty line.

Comment (ls-ggg) [reply src/lib.rs]: > Please remove this empty line.
@adrianreber 
Thanks. I have performed a rebase, removed the blank lines, and executed a cargo clippy check.

**Comment by adrianreber** [comment]
CI is still not happy. PTAL.

**Comment by ls-ggg** [comment]
> CI is still not happy. PTAL.

@adrianreber I have passed the checks of cargo fmt and cargo test. It should be okay now. Please run the CI again.

**Comment by ls-ggg** [comment]
> > CI is still not happy. PTAL.
> 
> @adrianreber I have passed the checks of cargo fmt and cargo test. It should be okay now. Please run the CI again.

and can the CI be set to trigger automatically?

**Pull request merged**